### PR TITLE
Add OCaml to translation table

### PR DESCRIPTION
**Pull request by Shanethegamer**
Also a few other minor tweaks

### CODE REVIEW

**Review by aderepas — CHANGES_REQUESTED** [review]
Thanks a lot for that translation and those slight modifications.
Just one thing: S= "a constant" in Python doesn't work if I'm mistaken

File: web/front/js-json-extra/translation.json [hunk 1]
```diff
@@ -173,7 +173,35 @@
...
+        "Type of the %" : "Truncated",
+        "Top level constructor" : false,
+        "Get a line of STDIN" : "read_line()",
+        "Get all STDIN" : false,
+        "Get a char STDIN" : "input_char stdin",
+        "Print without a new line at the end" : "print_string s",
+        "Print with a new line at the end" : "print_endline s",
+        "Implicit conversion of bool to int" : false,
+        "Implicit conversion of char to int" : false,
+        "Format a string" : "Printf.printf \"%d\" n",
```
Comment (Steffan153) [inline web/front/js-json-extra/translation.json:200]:
```suggestion
        "Format a string" : "Printf.sprintf \"%d\" n",
```
To keep it consistent with the rest.

**Comment by aderepas** [comment]
Seems ok!! 
Thank you for your contribution :D

File: web/front/js-json-extra/translation.json [hunk 2]
```diff
@@ -146,34 +146,62 @@
...
+        "Lambda declaration" : "a_lambda = fun argument -> argument",
+        "While loop" : "while b do\n\t...\ndone",
+        "For loop" : "for i = 0 to n - 1 do\n\t...\ndone",
+        "Foreach loop" : "for elem in a_list do\n\t...\ndone",
+        "Do .. while loop" : false,
+        "Foreach index and element loop" : false,
+        "Infinite loop" : "while true do\n\t...\ndone",
+        "if ... else if ... else" : "if b1 then\n\t...\nelse if b2 then\n\t...\nelse\n\t...",
+        "Ternary operator" : "if b then _true else _false",
+        "Try ... Catch" : "try \n\t...\ncatch e\n\t...\nend",
```
Comment (Steffan153) [inline web/front/js-json-extra/translation.json:189]: This is incorrect, it's `try ... with e -> ...`. Oh well, I'll fix it in my next PR.

**Pull request merged**